packages/core/core/src/utils/transform-content-types-to-models.ts

```diff
@@ -158,6 +158,13 @@ export const transformAttribute = (
         },
       };
     }
+    case 'relation': {
+      if (attribute.required) {
```

Comment:
I don't think we should prevent this here, (not the right place and considering we want to support it a bit later during RC if possible I don't see the point of make it a schema breaking change)

Reply:
I thought we wanted it to be not allowed 🤔 

In that case, instead of throwing an error I guess makes sense remove the required attribute in this same place in case if exists so it's not considered in the model.

Worth mention that in case you make a change in the CTB the required disappear because is not sent to the backend

Reply:
I think it's fine just not applying a required in the validator only for now until we can support it (target is GA for that)